### PR TITLE
Add addPolicy command

### PR DESCRIPTION
**Pull request by yorklim**
Client policies are not tracked 

Tracking client's policies will allow user to track the client's existing policies. Relates to #44 

Let's
- Add PolicyList, Policy, PolicyType to model
- Enable storage to handle policylist
- Users are to add Policy separately through the addpolicy command
- Update all data for testing and sample data
- Update all test cases for existing files and update for new classes

Follow Up
- Show PolicyList in GUI #67

### CODE REVIEW

File: src/main/java/seedu/address/model/policy/PolicyType.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+enum PolicyType {
+    HEALTH,
+    PERSONAL_ACCIDENT,
+    LIFE,
+    TRAVEL,
+    HOME,
+    CAR,
+    SAVING,
+    DEFAULT
+}
```
Comment (ReganChoy) [inline src/main/java/seedu/address/model/policy/PolicyType.java:12]: Good use of enums

File: src/main/java/seedu/address/model/policy/Policy.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    /**
+     * The Policy name.
+     */
+    public final String policyName;
+    /**
+     * The Policy id.
+     */
+    public final String policyID;
+    private final PolicyType type;
+
```
Comment (ReganChoy) [inline src/main/java/seedu/address/model/policy/Policy.java:37]: When we expand the commands to include PolicyTypes, we can consider if users can choose the policytypes by

1)Numbers -> 1 = Default, 2 = Travel etc
2)Spelling the policytypes out as per enums
3)Shortforms

If we are to use shortform or spelling, we should think if we should allow small/block letters only or are mixed casing allowed like dEfAuLt.

**Review by ReganChoy — COMMENTED** [review]
LGTM

**Review by solomonng2001 — APPROVED** [review]

**Comment by getsquared** [comment]
Closes #44

**Pull request merged**